rcl/test/rcl/test_time.cpp

```diff
@@ -263,11 +263,22 @@ TEST(CLASSNAME(rcl_time, RMW_IMPLEMENTATION), specific_clock_instantiation) {
   rcl_allocator_t allocator = rcl_get_default_allocator();
   {
     rcl_clock_t uninitialized_clock;
-    rcl_ret_t ret = rcl_clock_init(
-      RCL_CLOCK_UNINITIALIZED, &uninitialized_clock, &allocator);
+    rcl_ret_t ret = rcl_clock_init(RCL_CLOCK_UNINITIALIZED, &uninitialized_clock, &allocator);
```

Comment:
why the style change?

Reply:
It matches the other `rcl_clock_init` calls, and does not trigger linter errors